### DISCUSSION TITLE
problem lighting on surface in cest to night

### DISCUSSION THREAD

**Discussion by Cataldo90** [crest-ocean/crest#1082] [General]
i have a problem when to change weather in day to night, ocean crest is most illuminated , i have use crest lastest version and enviro sky, help me!!
![
![Cattura](https://user-images.githubusercontent.com/63008347/195576364-3191fe94-ed0a-4377-a0b8-e83b7210ab80.JPG)

**Comment by nitroman123** [comment]
you need to make a lot of changes to the ocean material in order to work try from there
